main.cpp

```diff
@@ -64,10 +64,16 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
-    
+    int speed = 30, volume = 10, height = 3;
+    float length = 1.52f, width = 2.33f, depth = 1.14f;
+    double pi = 3.14, constant = 5.5, voltage = 2.2;
+    bool jobFinished = 1, isColorWhite = 1, isShapeSquare = 0;
+    char mathGrade = 70, physicGrade = 80, chemistryGrade = 70; 
+    unsigned int amount = 10, month = 4, date = 5;
+
 
     
-    ignoreUnused(number); //passing each variable declared to the ignoreUnused() function
+    ignoreUnused(number,speed,volume,height,length,width,depth,pi,constant,voltage,jobFinished,isColorWhite,isShapeSquare,mathGrade,physicGrade,chemistryGrade, amount,month,date); //passing each variable declared to the ignoreUnused() function

```

Comment:
Add spaces after `,` to make the code more readable.
You did this for all of your variables declared above.
you should do it here, when passing arguments to functions as well
- [ ] revise